commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
```

Comment:
我们要加个自定义的. 原来的conventional默认set里面有常见的 feat, fix, doc什么的. 我们加一个 `archive: ` 用来表示提交文章的

Reply:
@boyangwang 
```suggestion
  extends: ['@commitlint/config-conventional'],
  rules: {
    'type-enum': [
      2,
      'always',
      [
        'build',
        'chore',
        'ci',
        'docs',
        'feat',
        'fix',
        'perf',
        'refactor',
        'revert',
        'style',
        'test',
        'archive',
      ],
    ],
  },0
```
添加 archive 属性了